src/System.Windows.Forms.Primitives/src/Interop/OleAut32/Interop.GetErrorInfo.cs

```diff
@@ -8,7 +8,7 @@ internal partial class Interop
 {
     internal static partial class Oleaut32
     {
-        [DllImport(Libraries.Oleaut32, ExactSpelling = true, PreserveSig = false)]
-        public static extern HRESULT GetErrorInfo(uint dwReserved, ref IErrorInfo pperrinfo);
+        [DllImport(Libraries.Oleaut32, ExactSpelling = true)]
+        public static extern void GetErrorInfo(uint dwReserved, out IErrorInfo pperrinfo);
```

Comment:
```suggestion
        public static extern HRESULT GetErrorInfo(uint dwReserved, out IErrorInfo pperrinfo);
```
The return value in the signature shoud match even if it is ignored.